### ISSUE TITLE
MSTest project type not yet supported

### ISSUE THREAD

**Issue by TheCakeMonster** [dotnet/upgrade-assistant#997]
Problem encountered on https://dotnet.microsoft.com/en-us/platform/upgrade-assistant/tutorial/install-upgrade-assistant
Operating System: windows

Version: 0.3.261602+efb3cc30a24e699e6e4337f263a2b9478f04f618

I tried to upgrade a .NET Framework unit test project using MSTest to the new SDK-style format using the latest version of upgrade-assistant today and the request to perform the upgrade was rejected because the project type is not yet supported.

It would be great if this additional project type were able to be supported by the upgrade-assistant global tool, as this would save people who are migrating old .NET Framework test projects to support .NET Core a lot of time.

**Comment by twsouthwick** [comment]
Can you provide a sample of what the project file looks like?

**Comment by twsouthwick** [comment]
Possibly related: https://github.com/dotnet/try-convert/issues/421

**Comment by TheCakeMonster** [comment]
> Can you provide a sample of what the project file looks like?

Yes, of course. Sorry for the delay. It's on GitHub at the following location. (This is the exact commit I tried it on, just in case we change this file in the next few days)

https://github.com/MarimerLLC/csla/blob/a050eb789bf92ed8acc23abe287f2704922d8d79/Source/Csla.test/csla.test.csproj

**Comment by sla89** [comment]
Same issue here but we are using Is there any workaround available until this is fixed?

**Comment by cliffAtOsaic** [comment]
+1 is there a workaround?

**Comment by dom-gibson-eso** [comment]
Good to see this has been sitting idle for 3 years...

**Comment by Japollack** [comment]
> Same issue here - is there any workaround available until this is fixed? Thank you :)

As a workaround remove '**{3AC096D0-A1C2-E12C-1390-A8335801FDAB};**' from ProjectTypeGuids:
`<ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>`
and it will convert just fine.

However it will not remove outdated references to Microsoft.VisualStudio.TestTools, Microsoft.VisualStudio.QualityTools.UnitTestFramework etc. but it can be done manually.
